packages/website/pages/pricing.js

```diff
@@ -1,34 +1,405 @@
+            </div>
+            <div id="intro_1-triangle_right">
+              <Img alt="" src={ImageTriangle} />
+            </div>
+            <div id="intro_1-cross_right">
+              <Img alt="" src={ImageCross} />
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </>
+);
 
-export default function Home() {
-  const sections = TiersPageData.page_content;
-  const animations = TiersPageData.floater_animations;
-
-  useEffect(() => {
-    let pageFloaters = {};
-    if (animations) {
-      initFloaterAnimations(animations).then(result => {
-        pageFloaters = result;
-      });
-    }
-    return () => {
-      if (pageFloaters.hasOwnProperty('destroy')) {
-        pageFloaters.destroy();
-      }
-    };
-  }, [animations]);
+const Card = props => {
+  const { title, price, isBestValue, children, storageAllocation, storageOverageRate, callToAction, callToActionUrl } =
+    props;
+  return (
+    <div className="pricing-card">
+      {!!isBestValue && <div className="best-value-adornment">Best Value!</div>}
+      <div className={`pricing-card-body${isBestValue ? ' adorned' : ''}`}>
+        <div className="pricing-title-row">
+          <div className="pricing-title">{title}</div>
+          <div className="pricing-price">
+            {price}
+            <div className="pricing-price-term">/month</div>
+          </div>
+        </div>
+        <ul className="plan-details">{children}</ul>
+        <div className="plan-summary">
+          <div className="plan-storage-allocation">{storageAllocation}</div>
+          <div className="plan-overage-rate">{storageOverageRate}</div>
+          <a href={callToActionUrl}>
+            <div className="plan-call-to-action button dark Button">{callToAction}</div>
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const PricingTiers = () => (
+  <>
+    <section id="section_plan_cards" className="sectional">
+      <div className="grid-middle">
+        <div className="col-12_sm-8_mi-10_ti-12 column-1" data-push-left="off-0_sm-2_mi-1_ti-0">
+          <div className="column-content">
+            <Card
+              title="Free"
+              price="$0"
+              storageAllocation="5GiB storage"
+              storageOverageRate=""
+              callToAction="GET STARTED"
+              callToActionUrl="/account/payment?plan=free"
+            >
+              <li className="pricing-bullet-1">Easily store your data and make it available on IPFS</li>
+              <li className="pricing-bullet-2">
+                All data is replicated onto the Filecoin storage network for verifiability that your data is safe
+              </li>
+              <li className="pricing-bullet-3">
+                Use the platform&apos;s other services like w3name and w3link to build the next generation of apps
+              </li>
+            </Card>
+
+            <Card
+              title="Lite"
```

Comment:
Yeah, let's get all this from plansContext. I think you can just `map` over the values of those plan objects and render the same component for each, and this'll shrink down nicely. can do later tho on first update of things

Reply:
I'll hold off doing it for now.  I'm not sure how that'll work with the bullet points.  Maybe those just need to be pushed up into the plansContext?